web-client/src/components/MainLayout/index.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { ReactNode } from 'react';
+import { NavLink, NavLinkLogo } from './styled';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import CssBaseline from '@mui/material/CssBaseline';
+import Divider from '@mui/material/Divider';
+import Drawer from '@mui/material/Drawer';
+import IconButton from '@mui/material/IconButton';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import MenuIcon from '@mui/icons-material/Menu';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import Grid from '@mui/material/Grid';
+
+import { appRoutesMap } from './../../routes/appRotesMap';
```

Comment:
I plan to connect ESlint and Prettier and configure them for the team working, it will be possible to configure there also absolute paths import. Ok, I'll do it right now, before the project gets big